src/main/java/com/example/ecommerce/dao/OrderRepository.java

```diff
@@ -4,6 +4,9 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface OrderRepository extends JpaRepository<Order, Long> {
+    List<Order> findByUser_Id(Long userId);
```

Comment:
```suggestion
    List<Order> findByUserId(Long userId);
```
đừng thêm `_` vào cu nhé, method và variable thì cứ dùng camel case đi nhé